jspwiki-main/src/main/java/org/apache/wiki/auth/DefaultUserManager.java

```diff
@@ -292,10 +293,10 @@ public UserProfile parseProfile( final Context context ) {
         String password = request.getParameter( PARAM_PASSWORD );
         String fullname = request.getParameter( PARAM_FULLNAME );
         String email = request.getParameter( PARAM_EMAIL );
-        loginName = InputValidator.isBlank( loginName ) ? null : loginName;
+        loginName = InputValidator.isBlank( loginName ) ? null : loginName.trim();
```

Comment:
`InputValidator.isBlank( loginName ) ? null : loginName.trim();` can be reduced to (commons-lang3)`StringUtils.trim( loginName )`, and same for the other two fields.